spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it "is valid with username and password" do
+    user_one = User.create(username: 'rossedfort', password: 'password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.